apps/react-remix/app/slash-id.tsx

```diff
@@ -12,7 +12,7 @@ export const SlashID = ({ children }: Props) => {
   return (
     <SlashIDProvider
       baseApiUrl="https://api.sandbox.slashid.com"
-      oid={process.env.SLASHID_ORGANIZATION_ID ?? ""}
+      oid="ORG_ID"
```

Comment:
I have no idea how this worked previously, as Remix does not expose env vars to the client (unlike Next.js)

Reply:
It did not work :)